### PR TITLE
data:Chandupuram

### PR DESCRIPTION
**Pull request by Chandupuram**
## To create an account with LinkFree
Added all the social media links to my accounts with correct icons as mentioned in docs.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4405"></a>

### CODE REVIEW

**Review by SaraJaoude — APPROVED** [review]
looks good

File: data/Chandupuram.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Puram Chandra Sekhar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Newbie in the community and interested in **Collab and Learn** "
```
Comment (swagataroy30) [inline data/Chandupuram.json:5]:
```suggestion
  "bio": "Newbie in the community and interested in **Collab and Learn** ",
```

**Review by swagataroy30 — COMMENTED** [review]
You need to add a comma at line 5. Thank You!

File: data/Chandupuram.json (hunk 1 above)
Comment (SaraJaoude) [inline data/Chandupuram.json:5]:
```suggestion
  "bio": "Newbie in the community and interested in **Collab and Learn**",
```

**Review by sidharthviz — CHANGES_REQUESTED** [review]
Hi @Chandupuram  Please add the comma in line no. 5 
Use this for better reference  <https://jsonlint.com/>

**Pull request merged**